### PR TITLE
Fix duplication of GENERIC_READ & GENERIC_WRITE

### LINKED ISSUE #42199: Remove duplicates of Interop.Kernel32.GenericOperations.GENERIC_READ and GENERIC_WRITE
There are a few places in corefx that are declaring these two constants, but we should be consuming them solely from Interop.Kernel32.GenericOperations.
A few of them are shared with CoreLib so the file needs to be moved to the correct location.
See this comment for more context:
https://github.com/dotnet/corefx/pull/42099#discussion_r339838409

### PR DESCRIPTION
**Pull request by Youssef1313**
Fixes #42199 (Hope my changes are correct)

### CODE REVIEW

**Comment by Youssef1313** [comment]
/cc @carlossanlop @jkotas
Can you review the changes here ?
I'd also like to know if I should open a new PR when the repos are merged to dotnet/runtime ?

File: src/Common/src/CoreLib/System.Private.CoreLib.Shared.projitems [hunk 1]
```diff
@@ -1055,6 +1055,7 @@
     <Compile Include="$(MSBuildThisFileDirectory)Interop\Windows\Kernel32\Interop.FindClose.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)Interop\Windows\Kernel32\Interop.FindFirstFileEx.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)Interop\Windows\Kernel32\Interop.FlushFileBuffers.cs" />
+	<Compile Include="$(MSBuildThisFileDirectory)Interop\Windows\Kernel32\Interop.GenericOperations.cs" />
```
Comment (jkotas) [inline src/Common/src/CoreLib/System.Private.CoreLib.Shared.projitems]:
```suggestion
	  <Compile Include="$(MSBuildThisFileDirectory)Interop\Windows\Kernel32\Interop.GenericOperations.cs" />
```

**Comment by carlossanlop** [comment]
The change looks good, @Youssef1313 , although I don't know what the process would be to get this ported to dotnet/runtime. @jkotas would you mind chiming in here?

**Comment by jkotas** [comment]
See the boilerplate comment in older PRs, e.g.: https://github.com/dotnet/corefx/pull/41885#issuecomment-550501464

**Comment by Youssef1313** [comment]
@jkotas, Thanks for clarification. I'll wait for the dotnet/runtime to be public and open a PR there.

**Comment by carlossanlop** [comment]
Thanks @jkotas.
@Youssef1313 ping me when you submit this same change in the dotnet/runtime repo, and I'll sign it off. Thanks for working on this.

**Comment by carlossanlop** [comment]
@Youssef1313 FYI the dotnet/runtime repo has been opened to the public. Please open a new PR there with these changes.

**Comment by Youssef1313** [comment]
@carlossanlop, Thanks for notifying me.
Unfortunately, I'm not sure why I'm not able to fork dotnet/runtime.

image

**Comment by carlossanlop** [comment]
> _Unfortunately, I'm not sure why I'm not able to fork dotnet/runtime._

Thanks for letting me know. Let me notify my team about your issue and I'll come back to you.

**Comment by Youssef1313** [comment]
@carlossanlop, I was able to clone it locally by specifying `--depth 1` in the clone command, then fetch with some depth and gradually increase it. The problem now is:

image

and it's because the `Common` resides in `src/libraries` not `src/libraries/src`. Not sure how to get into that.

/cc @maryamariyan

***

**Update:** I've overcome this problem by editing the patch files manually, but got some other problems due to folder layout changes after consolidation, so I'm going to re-make these changes manually.

**Comment by Youssef1313** [comment]
Opened a PR in runtime repo. Closing this.

**Pull request closed without merging**